lib/utils/matchUrl.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var { MatchPattern } = require("sdk/util/match-pattern");
+
+/**
+ * Check if URI is in whitelist
+ * @param include object {0: "*", ...}
+ * @param uri to test
+ * @return testResult true or false
+ */
+exports.isUriIncluded = function(include, uri) {
```

Comment:
The variable name should convey that it is a list of some sort (which it is, isn't it?). See e.g. here for discussion about naming: http://stackoverflow.com/questions/395739/do-you-name-your-arrays-plurally-or-singularly, They would name it includePatterns or includePatternList,